Java/JDI/jdi-cucumber-test-generator/src/main/java/com/epam/test_generator/controllers/AutoCompleteController.java

```diff
@@ -0,0 +1,51 @@
+package com.epam.test_generator.controllers;
+
+import com.epam.test_generator.dto.AutoCompleteDTO;
+import com.epam.test_generator.services.AutoCompleteService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Controller
+public class AutoCompleteController {
+
+    @Autowired
+    AutoCompleteService autoCompleteService;
+
+    @RequestMapping(value = "/manageSteps")
+    public String getMainPage() {
+        return "/WEB-INF/static/views/manageSteps";
+    }
+
+    @RequestMapping(value = "/getAutoCompleteList", method = RequestMethod.GET, produces = "application/json")
```

Comment:
Let's stick to the REST way of building paths. Something like /step/autocomplite